scripts/doctor

```diff
@@ -42,7 +42,16 @@ else
     count_in_path "bundler"
```

Comment:
```suggestion
    count_in_path "bundle"
```

Lets just count_in_path `bundle` instead of `bundler` and omit the other changes. If you want you can retain the warning for multiple PATH references.